app/Policies/UserPolicy.php

```diff
@@ -10,8 +10,8 @@ class UserPolicy
 	use HandlesAuthorization;
 
 	public const IS_ADMIN = 'isAdmin';
-	public const CAN_UPLOAD = 'canUpload';
-	public const CAN_EDIT_SETTINGS = 'canEditSettings';
+	public const MAY_UPLOAD = 'mayUpload';
```

Comment:
`MAY_UPLOAD` should not exist at all. A policy returns _rights_ not capabilities. Moreover, one cannot upload a user.